frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -483,6 +550,35 @@ function ReaderHighlight:addToMainMenu(menu_items)
             })
         end,
     })
+    table.insert(menu_items.highlight_options.sub_item_table, {
+        text = _("Apply default style to all highlights"),
+        callback = function(touchmenu_instance)
+            UIManager:show(ConfirmBox:new{
+                text = _("Are you sure you want to update all highlights?"),
+                icon = "texture-box",
+                ok_callback = function()
+                    local count = 0
+                    for _, items in pairs(self.view.highlight.saved) do
```

Comment:
Deprecated highlights table.

Reply:
What has it been replaced with?

Reply:
Something like `for _, item in ipairs(self.ui.annotation.annotations) do`, right?

Reply:
`self.ui.annotation.annotations`